<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportTariff_Wizard_1</class>
 <widget class="QDialog" name="ExportTariff_Wizard_1">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор экспортируемых тарифов</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="chkExportAll">
     <property name="text">
      <string>Выгружать всё</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="CInDocTableView" name="tblItems">
     <property name="whatsThis">
      <string comment="ура!">список записей</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>229</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnSelectAll">
     <property name="text">
      <string>Выбрать все</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btnClearSelection">
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkExportAll</tabstop>
  <tabstop>tblItems</tabstop>
  <tabstop>btnSelectAll</tabstop>
  <tabstop>btnClearSelection</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
